### PR TITLE
Add puppet-strings docs

### PR DESCRIPTION
**Pull request by baurmatt**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1,86 +1,61 @@
...
 #  class {
 #    'dropbear':
 #      port            => 443,
 #      extra_args      => '-s',
 #      banner          => '/etc/banner',
 #  }
-#
-# === Authors
-#
-# Kyle Anderson <kyle@xkyle.com>
```
Comment (alexjfisher) [inline manifests/init.pp:76]: I *think* there's an `@author` tag.  I'm not against leaving these in if that's the case.

File: manifests/init.pp [hunk 2]
```diff
@@ -1,86 +1,61 @@
...
 #  include 'dropbear'
 #
-#   or
-#
+# @example
 #  class {
 #    'dropbear':
 #      port            => 443,
 #      extra_args      => '-s',
 #      banner          => '/etc/banner',
```
Comment (alexjfisher) [inline manifests/init.pp:57]: Remove the random indentation here?

**Review by alexjfisher — CHANGES_REQUESTED** [review]
Looks good.  The example could do with a minor reformatting, and could you turn on puppet-lint-param-docs?

File: manifests/init.pp (hunk 1 above)
Comment (baurmatt) [reply manifests/init.pp:76]: As discussed in another ticket, yes there is! :) Changed! :)

**Comment by baurmatt** [comment]
Added the @author tag and `puppet-lint-param-docs`! :)

**Review by alexjfisher — APPROVED** [review]

**Pull request merged**